### PR TITLE
Add in fixes to object storage threats, controls, features

### PR DESCRIPTION
**Pull request by mlysaght2017**

### CODE REVIEW

File: services/common-controls.yaml [hunk 1]
```diff
@@ -201,7 +201,7 @@ controls:
       reduce exposure to geopolitical risks.
     control_family: Data
     threats:
-      - CCC.TH05
+      - CCC.TH03
```
Comment (eddie-knight) [inline services/common-controls.yaml:204]:
```suggestion
      - CCC.TH03 # Deployment region network is untrusted
```

File: services/common-controls.yaml [hunk 2]
```diff
@@ -283,6 +284,54 @@ controls:
...
+        tlp_levels:
+          - tlp_green
+          - tlp_amber
+          - tlp_red
+  
+  - id: CCC.C09
+    title: Prevent tampering, deletion, or unauthorized access to access logs
+    control_family: Data
+    objective: |
+      Ensure that access logs are protected against unauthorized access, tampering,
```
Comment (eddie-knight) [inline services/common-controls.yaml:296]:
```suggestion
      Access logs should always be considered sensitive.
      Ensure that access logs are protected against unauthorized access, tampering, 
```

File: services/common-controls.yaml [hunk 3]
```diff
@@ -283,6 +284,54 @@ controls:
...
+      or deletion.
+    threats:
+      - CCC.TH07 # Logs are Tampered With or Deleted
+      - CCC.TH09 # Logs or Monitoring Data are Read by Unauthorized Users
+      - CCC.TH04 # Resource is replicated to untrusted or external locations
+    nist_csf: PR.DS-6 # Integrity checking mechanisms are used to verify software, firmware, and information integrity
+    test_requirements:
+      - id: CCC.C09.TR01
+        text: |
+          Access logs cannot be accessed, tampered with, or deleted by unauthorized users.
```
Comment (eddie-knight) [inline services/common-controls.yaml:306]:
```suggestion
          Access logs cannot be accessed without proper authorization.
        tlp_levels:
          - tlp_amber
          - tlp_red
          - tlp_green
          - tlp_clear
      - id: CCC.C09.TR02
        text: |
          Access logs cannot be modified without proper authorization.
        tlp_levels:
          - tlp_amber
          - tlp_red
          - tlp_green
          - tlp_clear
      - id: CCC.C09.TR03
        text: |
          Access logs cannot be deleted without proper authorization.
        tlp_levels:
          - tlp_amber
          - tlp_red
          - tlp_green
          - tlp_clear
```

File: services/common-controls.yaml [hunk 4]
```diff
@@ -283,6 +284,54 @@ controls:
...
+        tlp_levels:
+          - tlp_amber
+          - tlp_red
+      - id: CCC.C09.TR02
+        text: |
+          Access logs cannot be replicated to untrusted or external locations.
+        tlp_levels:
+          - tlp_amber
+          - tlp_red
```
Comment (eddie-knight) [inline services/common-controls.yaml:315]:
```suggestion
```

**Review by eddie-knight — APPROVED** [review]

**Review by damienjburks — DISMISSED** [review]

**Comment by mlysaght2017** [comment]
@eddie-knight @damienjburks @smendis-scottlogic - I believe this is now ready to merge. Can you please do the necessary.

**Review by damienjburks — APPROVED** [review]

**Review by smendis-scottlogic — APPROVED** [review]

**Pull request merged**